cabal-fmt.sh

```diff
@@ -1 +1 @@
-find . -name '*.cabal' -exec sh -c 'cabal-fmt $0 > output.tmp; mv output.tmp $0' {} ';'
+find . -name '*.cabal' -exec sh -c 'cabal-fmt -i $0' {} ';'
```

Comment:
@rachelcarmena here's the improved version! 😉 thanks a lot for your help! ❤️

Reply:
Thanks to you for letting me know about it :heart: @serras another :heart: for you, as well :sweat_smile: